flybywiresim/a32nx
# fix: Prevent TCAS STBY warning from appearing in wrong flight phase

Fixes #3490

## Summary of Changes


Fixes a missing explicit flight phase check from #3378. This prevents the message from spuriously being triggered on ground due to a flight phase change.

In addition I've removed it from the STS page, because after researching this more I believe there is no possible way that NAV TCAS STBY and STS TCAS can appear at the same time, due to the former TCAS FAULT to be false and the latter requiring TCAS FAULT to be true.

Not adding to CHANGELOG as I'm fixing something minor that was only added this release and already has a CHANGELOG entry.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): BehEh#1234

## Testing instructions


1. Spawn cold and dark
2. Ensure the TCAS switch is set to STBY
3. Observe no amber NAV TCAS STBY warning, only the green TCAS STBY memo (on the right)
4. Take off with TCAS remaining in STBY
5. Upon T.O INHIBIT disappearing, observe:  
NAV TCAS STBY warning in amber
TCAS STBY memo turning amber
6. Turn the ADIRS off and observe NAV TCAS STBYs disappearing and NAV TCAS FAULT appearing (TCAS STBY memo remains amber)
7. TCAS should appear on the STATUS page only together with TCAS FAULT (remains when clearing), but never together with NAV TCAS STBY


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
